dev/wlp-jakartaee-transform/rules/jakarta-faces-initialization.properties

```diff
@@ -1,7 +1,4 @@
 #com.ibm.ws.myfaces.initializer.xml
 javax.servlet.ServletContainerInitializer=jakarta.servlet.ServletContainerInitializer
-#com.ibm.ws.org.apache.myfaces_3_0_1_0.ResourceProvider.xml
+#com.ibm.ws.org.apache.myfaces_1_0.ResourceProvider.xml
 META-INF/services/javax.faces.application.ApplicationFactory=META-INF/services/jakarta.faces.application.ApplicationFactory
-#com.ibm.ws.jsf_3_0.cdiextension.xml
-javax.faces.flow.FlowScoped;javax.faces.view.ViewScoped=jakarta.faces.flow.FlowScoped;jakarta.faces.view.ViewScoped
-javax.faces.flow.builder.FlowDefinition;javax.faces.flow.builder.FlowBuilderParameter=jakarta.faces.flow.builder.FlowDefinition;jakarta.faces.flow.builder.FlowBuilderParameter
```

Comment:
Removed since these are not needed for transformation

Reply:
Any idea why these were added if they are not necessary? Or have you just done some additional testing and it is a mistake?

Reply:
Place by mistake. Each MyFaces Impl bundle has it's own respective *cdiextension.xml.  The file is already correctly  configured, and the the transformation didn't do anything. 

https://github.com/OpenLiberty/open-liberty/blob/a8ec30f99203ca7696c97b31b6061603590bf597/dev/io.openliberty.org.apache.myfaces.3.0/bnd.bnd#L37-L43

Speaking of this bundle, I'll get another PR to use the io.openliberty.* prefix for these files in 3.0 impl bundle.

Issue here ->  [#16975](https://github.com/OpenLiberty/open-liberty/issues/16975)